chrome/popup.js

```
@@ -0,0 +1,279 @@
+  
+  // Load ban list from content script
+  const loadBanList = async () => {
+    if (!isYouTube) {
+      updateStatus('Please open a YouTube page to use this extension', 'error');
+      return;
+    }
+    
+    try {
+      const response = await chrome.runtime.sendMessage({ action: 'getBanListFromPage' });
+      
+      if (response && response.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)